### ISSUE TITLE
will you release the training code for posetrack18?

### ISSUE THREAD

**Issue by aaab8b** [Guanghan/lighttrack#10]
if i want to reproduce your result on posetrack18, is there anything i should notice about?

**Comment by aaab8b** [comment]
should i use official eval code to convert posetrack18 gt_annotations to posetrack 17 format and then use merge_PoseTrack_json.py? is all the other parameters the same?

**Comment by Guanghan** [comment]
The model is trained with PoseTrack'17 and COCO, because it is for the testing on PoseTrack'17 for fair comparison. You can follow the README.md to easily re-produce the results on PoseTrack'18 validation set with the provided model. 

If you want to train with PoseTrack'18, you can perform some tweaks on the code https://github.com/Guanghan/lighttrack/blob/master/HPE/AllJoints_PoseTrack.py. 
You can just concatenate the PoseTrack'18 annotations to get the merged json file; no need to transform it to 17 format. Just make sure the annotations can be successfully read by the data feeder.
